### DISCUSSION TITLE
Netflix buffering

### DISCUSSION THREAD

**Discussion by frownlayer** [CastagnaIT/plugin.video.netflix#1014] [Q&A] [answered]
Excuse bad English.

Raspberry pi 4 4gb, libreelec, kodi 18.9, netflix 1.12.0.

My downloadspeed is catastrophic low. I have 100mbps to the computer, and there's no problem. But when I stream from Netflix its just buffering all the time.

I have a video that may help you understand.
https://youtu.be/X3xHbAhqUnw

**Comment by CastagnaIT** [comment]
Wiki:
https://github.com/CastagnaIT/plugin.video.netflix/wiki/FAQ-(Audio,-Video,-Subtitle,-Other)#how-to-try-solve-the-problem-with-loading-screen-video-buffering

**Reply by frownlayer** [reply]
I'm down to 400000, 480p, and my raspberry has ethernet connection. But it still buffering all the time

**Comment by CastagnaIT** [comment, marked as answer]
in these days netflix servers works very bad, you can try change CDN server:
open `Expert` settings and try change `Open Connect CDN` server, if no improvement remember to restore the `Server 1`
